### PR TITLE
Fixed: Manual Import with multiple languages

### PR DESCRIPTION
**Pull request by danimart1991**
#### Database Migration
NO

#### Description
Fix Fatal Error: "_Request Failed. POST /api/v3/manualimport: Sequence contains more than one element" when trying to manual import a movie with more than one language._"

Exception:
```
System.InvalidOperationException: Sequence contains more than one element
   at System.Linq.ThrowHelper.ThrowMoreThanOneElementException()
   at System.Linq.Enumerable.TryGetSingle[TSource](IEnumerable`1 source, Boolean& found)
   at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source)
   at NzbDrone.Core.MediaFiles.MovieImport.Manual.ManualImportService.ReprocessItem(String path, String downloadId, Int32 movieId, String releaseGroup, QualityModel quality, List`1 languages) in D:\a\1\s\src\NzbDrone.Core\MediaFiles\MovieImport\Manual\ManualImportService.cs:line 111
   at Radarr.Api.V3.ManualImport.ManualImportController.ReprocessItems(List`1 items) in D:\a\1\s\src\Radarr.Api.V3\ManualImport\ManualImportController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Radarr.Http.Middleware.BufferingMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\BufferingMiddleware.cs:line 28
   at Radarr.Http.Middleware.IfModifiedMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\IfModifiedMiddleware.cs:line 41
   at Radarr.Http.Middleware.CacheHeaderMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\CacheHeaderMiddleware.cs:line 33
   at Radarr.Http.Middleware.UrlBaseMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\UrlBaseMiddleware.cs:line 27
   at Radarr.Http.Middleware.VersionMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\VersionMiddleware.cs:line 28
   at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionMiddleware.InvokeCore(HttpContext context)
   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
```

### CODE REVIEW

**Comment by bakerboy448** [comment]
More so out of curiosity - what's reporting to radarr the multiple languages?

Those typically should be getting treated as multi then,  but someone may have been regressed along the way

**Comment by danimart1991** [comment]
The behavior:

As I got the movie file in another pc. 
- I move the movie to: "Movie (2022)/Movie (2022) [ES-EN].mkv".
- I press the Manual Import button.
- Fill the Manual Import details including multilanguage.
- The movie imports.
- Then press Update button to generate extra files.
- Click rename to rename all files to the correct Radarr Media Management config.
- All ok.

Then I checked the events section to see that the error I posted happened.

**Comment by danimart1991** [comment]
Ready to review.

File: src/NzbDrone.Core/MediaFiles/MovieImport/Manual/ManualImportService.cs [hunk 1]
```diff
@@ -121,7 +121,7 @@ public ManualImportItem ReprocessItem(string path, string downloadId, int movieI
                 SceneSource = SceneSource(movie, rootFolder),
                 ExistingFile = movie.Path.IsParentPath(path),
                 Size = _diskProvider.GetFileSize(path),
-                Languages = (languages?.SingleOrDefault() ?? Language.Unknown) == Language.Unknown ? languageParse : languages,
```
Comment (Qstick) [inline src/NzbDrone.Core/MediaFiles/MovieImport/Manual/ManualImportService.cs:124]: We only want to enforce this logic if there is one language and it's unknown, thus the single. We need to fix this another way.

**Pull request closed without merging**